src/pages/ImportData.js

```diff
@@ -118,10 +135,37 @@ const ImportData = () => {
         notify('success', 'Data imported successfully.', { timeout: 3000 })
       }],
       ['snapshot', async () => {
-        snapshots && snapshots.length > 0 ?
-          await Promise.allSettled(_.map(({ title, id }) => Ajax().Workspaces.workspace(namespace, name).importSnapshot(id, title), snapshots)) :
+        if (snapshots?.length > 0) {
+          const responses = await Promise.allSettled(
+            _.map(({ title, id, description }) => {
+              // Normalize the title:
+              // Importing snapshot will throw an "enum" error if the title has any spaces or special characters
+              // Replace all whitespace characters with _
```

Comment:
nit could you remove the comments on line 143 and 144. The comment Mohs had earlier was more about adding the "why" and removing the "how" of normalize code.

Reply:
I honestly suck at reading regex, it's my preference to leave any comments that explain what the regex is targeting XD; Is that okay if we keep it?

Reply:
That works for me